pkg/paymasterclient/types.go

```diff
@@ -16,6 +16,27 @@ type TransactionArgs struct {
 	Data  *hexutil.Bytes  `json:"data"`
 }
 
+// IsSponsorableOptions defines the options for the IsSponsorable method.
+type IsSponsorableOptions struct {
+	// PrivatePolicyUUID is the UUID of a private policy.
+	// This field should only be set when using a sponsor client.
+	// When set, it allows the use of a private policy for the sponsorable check.
+	// For paymaster client calls, this field should be left empty.
+	PrivatePolicyUUID string
+}
+
+// SendRawTransactionOptions defines the options for the SendRawTransaction method.
+type SendRawTransactionOptions struct {
+	// PrivatePolicyUUID is the UUID of a private policy.
+	// This field should only be set when using a sponsor client.
+	// When set, it allows the use of a private policy for the transaction.
+	// For paymaster client calls, this field should be left empty.
+	PrivatePolicyUUID string
```

Comment:
for `paymasterclient` here, there is no need to have `PrivatePolicyUUID`, for `sponsorclient`, it must hav the `PrivatePolicyUUID`.

But it is optional here, will confuse the user who only use  `paymasterclient`